deps/chakrashim/src/v8handlescope.cc

```diff
@@ -23,13 +23,13 @@
 
 namespace v8 {
 
-__declspec(thread) HandleScope *current = nullptr;
+THREAD_LOCAL HandleScope *current = nullptr;
 
 HandleScope::HandleScope(Isolate* isolate)
-    : _prev(current),
-      _locals(),
+    : _locals(),
```

Comment:
Same as other cases, to align declaration and initialization order to make compiler happy.